pipeline/06-upload.R

```diff
@@ -30,7 +28,7 @@ if (upload_enable) {
   # Initialize a dictionary of paths AND S3 URIs specific to the run ID and year
   paths <- model_file_dict(
     run_id = run_id,
-    year = params$assessment$year
+    year = params$assessment$upload_year
```

Comment:
**[Nitpick, blocking]** Hm, maybe I'm missing something, but shouldn't this be `working_year`?

Reply:
Woops, you are completely right. Nice catch.